### PR TITLE
Add `process.runtime.jvm.cpu.time` metric

### PR DESCRIPTION
**Pull request by trask**
Fixes https://github.com/open-telemetry/opentelemetry-specification/issues/3490

Adds `process.runtime.jvm.cpu.time` metric.

### CODE REVIEW

File: semantic_conventions/metrics/process-runtime-jvm-metrics.yaml [hunk 1]
```diff
@@ -122,6 +122,13 @@ groups:
     instrument: updowncounter
     unit: "{class}"
 
+  - id: metric.process.runtime.jvm.cpu.time
+    type: metric
+    metric_name: process.runtime.jvm.cpu.time
+    brief: "Number of classes currently loaded."
```
[the remark this answers is not in the document]
Comment (trask) [reply semantic_conventions/metrics/process-runtime-jvm-metrics.yaml]: 🙈

**Review by mateuszrzeszutek — APPROVED** [review]

File: specification/metrics/semantic_conventions/runtime-environment-metrics.md [hunk 2]
```diff
@@ -297,6 +298,22 @@ This metric is obtained from [`ClassLoadingMXBean#getLoadedClassCount()`](https:
 <!-- semconv metric.process.runtime.jvm.classes.current_loaded(full) -->
 <!-- endsemconv -->
 
+### Metric: `process.runtime.jvm.cpu.time`
+
+This metric is [recommended][MetricRecommended].
+
+This metric is obtained from [`com.sun.management.OperatingSystemMXBean#getProcessCpuTime()`](https://docs.oracle.com/en/java/javase/17/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getProcessCpuTime()) on HotSpot
```
Comment (joaopgrassi) [inline specification/metrics/semantic_conventions/runtime-environment-metrics.md:305]: Wouldn't it be nicer if these are added as `note` in `yaml`, as done in https://github.com/open-telemetry/semantic-conventions/pull/13?

File: semantic_conventions/metrics/process-runtime-jvm-metrics.yaml [hunk 3]
```diff
@@ -122,6 +122,13 @@ groups:
     instrument: updowncounter
     unit: "{class}"
 
+  - id: metric.process.runtime.jvm.cpu.time
+    type: metric
+    metric_name: process.runtime.jvm.cpu.time
+    brief: "CPU time used by the process."
+    instrument: counter
+    unit: "s"
```
Comment (joaopgrassi) [inline semantic_conventions/metrics/process-runtime-jvm-metrics.yaml:130]: Looking at the links below https://docs.oracle.com/en/java/javase/17/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getProcessCpuTime() says

>  The returned value is of nanoseconds precision but not necessarily nanoseconds accuracy

I wonder if we should use `ns` here instead?

Comment (breedx-splk) [reply semantic_conventions/metrics/process-runtime-jvm-metrics.yaml:130]: Otel spec has mostly standardized on decimal seconds for time representations.

**Review by jack-berg — APPROVED** [review]

File: specification/metrics/semantic_conventions/runtime-environment-metrics.md (hunk 2 above)
Comment (joaopgrassi) [reply specification/metrics/semantic_conventions/runtime-environment-metrics.md:305]: I just noticed that `notes` for metrics are not being generated in the markdown files. They only seem to show up for `attributes`. I will investigate the build tools and open an issue if that's the case.

File: semantic_conventions/metrics/process-runtime-jvm-metrics.yaml [hunk 4]
```diff
@@ -122,6 +122,13 @@ groups:
     instrument: updowncounter
     unit: "{class}"
 
+  - id: metric.process.runtime.jvm.cpu.time
+    type: metric
+    metric_name: process.runtime.jvm.cpu.time
+    brief: "CPU time used by the process."
```
[the remark this answers is not in the document]
Comment (trask) [reply semantic_conventions/metrics/process-runtime-jvm-metrics.yaml:128]: I opened #65 for this, in order to apply this suggestion across the couple of the other process/system metrics reported under the JVM namespace

**Review by breedx-splk — APPROVED** [review]
Looks good to me too!

**Comment by trask** [comment]
@arminru @jsuereth @reyang do we need more approvals or is this good to merge? thx

**Review by reyang — APPROVED** [review]

**Comment by trask** [comment]
I will fix broken link in this PR (similar issue as #96)

**Review by joaopgrassi — APPROVED** [review]

**Pull request merged**